.github/workflows/release.yml

```diff
@@ -0,0 +1,91 @@
+name: CI Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
```

Comment:
So this action happens on every push?

I could be wrong but I dont recall tags having an associated push, which is why I read the trigger to be every push, or never.

Reply:
This action will be triggered when there is a push to a tag (creation of the tag)

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-branches-and-tags

I tested this on my fork and pushes to a branch didnt trigger the action, only creation of a tag (from creating a release)